module.txt

```diff
@@ -1,7 +1,7 @@
 {
     "id" : "SimpleFarming",
     "version" : "1.0.0-SNAPSHOT",
-    "author" : "Josharias, patrick Wang",
+    "author" : "Josharias, patrick Wang, SufurElite“,
```

Comment:
Watch out - you got the right kind of double quote added here, `“` instead of `"` - tiny difference, large impact, as the game actually fails to initialize with this in place! :-)

I hope you tested your change in-game and just made this change last, otherwise I would suspect it may mean you haven't tested at all ... ;-)